src/datasource.ts

```diff
@@ -92,9 +92,10 @@ export class GenericDatasource {
     });
   }
 
-  metricFindQuery(query) {
+  findMetricsQuery(type: string, query: string) {
```

Comment:
I'd invert arguments though. `query` should be first as it's more important than type. WDYT?

Reply:
Had the same feeling during writing, done.